From: Andrew Morton <akpm@linux-foundation.org>
Subject: maple_tree-reorganize-testing-to-restore-module-testing-fix
Date: Fri Oct 28 02:54:27 PM PDT 2022

fix module export

ERROR: modpost: "mas_find_rev" [lib/test_maple_tree.ko] undefined!

Cc: Liam Howlett <liam.howlett@oracle.com>
Signed-off-by: Andrew Morton <akpm@linux-foundation.org>
---

 lib/maple_tree.c |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/lib/maple_tree.c~maple_tree-reorganize-testing-to-restore-module-testing-fix
+++ a/lib/maple_tree.c
@@ -6059,7 +6059,7 @@ void *mas_find_rev(struct ma_state *mas,
 	/* Retries on dead nodes handled by mas_next_entry */
 	return mas_prev_entry(mas, min);
 }
-EXPORT_SYMBOL_GPL(mas_find);
+EXPORT_SYMBOL_GPL(mas_find_rev);
 
 /**
  * mas_erase() - Find the range in which index resides and erase the entire
_
